### PR TITLE
Add node affinity for don's JH pods so that they land on the v100 GPUs

### PR DESCRIPTION
**Pull request by maulikjs**

### CODE REVIEW

File: jupyterhub/bases/custom-profiles/jupyterhub-singleuser-profiles.yaml [hunk 1]
```diff
@@ -78,7 +78,16 @@ profiles:
     cpu_limit: 16
   env:
     ENABLE_HOST_IPC: true
-
+  # Set OpenShift node affinity for a notebook pod. See https://kubernetes.io/docs/concepts/configuration/assign-pod-node/ for more information.
+  node_affinity:
+  requiredDuringSchedulingIgnoredDuringExecution:
```
Comment (accorvin) [inline jupyterhub/bases/custom-profiles/jupyterhub-singleuser-profiles.yaml]: This block should be indented, shouldn't it?

File: jupyterhub/bases/custom-profiles/jupyterhub-singleuser-profiles.yaml [hunk 2]
```diff
@@ -78,7 +78,16 @@ profiles:
...
+    nodeSelectorTerms:
+      - matchExpressions:
+        - key: gpu_type
+          operator: In
+          values:
+          - GV100GL
```
Comment (accorvin) [inline jupyterhub/bases/custom-profiles/jupyterhub-singleuser-profiles.yaml]: This line, while valid yaml, should be indented for consistent formatting of lists

Comment (maulikjs) [reply jupyterhub/bases/custom-profiles/jupyterhub-singleuser-profiles.yaml]: done!

File: jupyterhub/bases/custom-profiles/jupyterhub-singleuser-profiles.yaml (hunk 1 above)
Comment (maulikjs) [reply jupyterhub/bases/custom-profiles/jupyterhub-singleuser-profiles.yaml]: yup! done!

**Review by accorvin — APPROVED** [review]
I can't comment on whether this actually works, I haven't tested it, but it looks good to me.

**Pull request merged**